bin/metababel

```diff
@@ -91,11 +91,10 @@ def wrote_event_dispatchers(folder, t)
   event_name = 'event'
   event_class_dispatchers = t.map_event_classes_with_index do |e, index_stream_class, index_event_class, default_clock_class|
     arg_variables = {}
-    arg_variables['outputs'] = [GeneratedArg.new('int64_t', 'timestamp')] if default_clock_class
     body = Babeltrace2Gen.context(indent: 1) do
       e.get_getter(event: event_name, arg_variables: arg_variables)
     end
-    EventClassInfo.new(e.name, arg_variables.fetch('outputs', []), "\n" + body, index_stream_class, index_event_class, default_clock_class,
+    EventClassInfo.new(e.name, arg_variables.fetch('outputs', []), body, index_stream_class, index_event_class, default_clock_class,
```

Comment:
Why remove the new line? I think it was required to get a nice code generation

Reply:
Agree, but now since we move the event out from paremeters and it is a variable, I need to check the body to see if we need to render the event getter for the fields in the body. With '\n' I have to check if the body is empty like this `body == '\n'`, I change it to do this `body.empty?`. I explain how I fix the rendering in the next comment.